src/platform/input/mouse-button.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Mouse buttons.
+ *
+ * @readonly
+ * @enum {ValueOf<typeof MouseButton>}
```

Comment:
Aha, no import for `ValueOf`? We either need to dev-install `type-fest` or just using the inline typedef:

```js
/** @template T @typedef {T[keyof T]} ValueOf */
```

I mean, technically we don't even need it at all, it just looks a bit nicer/shorter than this (which we could also use):

```js
@enum {typeof MouseButton[keyof typeof MouseButton]}
```